### PR TITLE
better support for networked python executable

### PR DESCRIPTION
**Pull request by tokejepsen**

### CODE REVIEW

File: pyblish_ftrack/ftrack_event_plugin_path/publish.py [hunk 1]
```diff
@@ -141,18 +141,31 @@ def _discoverApplications(self, path=''):
...
             label='Publish',
             applicationIdentifier='publish',
-            icon="https://raw.githubusercontent.com/pyblish/pyblish-ftrack/master/pyblish_ftrack/ftrack_event_plugin_path/icon.png",
+            icon=icon,
             launchArguments=launchArguments
         )
 
+        for path in os.environ['PATH'].split(os.pathsep):
+            if 'python' in path.lower():
+                path = os.path.join(path, 'python.exe')
```
Comment (mottosso) [inline pyblish_ftrack/ftrack_event_plugin_path/publish.py:157]: Not very Linux friendly :(

Comment (tokejepsen) [inline pyblish_ftrack/ftrack_event_plugin_path/publish.py:157]: only working with Windows atm, so will tackle linux when the need comes.

Comment (mkolar) [inline pyblish_ftrack/ftrack_event_plugin_path/publish.py:157]: same with us. Don't even have linux machines to test things. If someone PRs linux support I'll be more than happy. Until then I think we can merge this

Comment (tokejepsen) [inline pyblish_ftrack/ftrack_event_plugin_path/publish.py:157]: For other OSs you would just query which OS you are in, and implement whatever logic there is to find the executable.

**Pull request merged**